<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateOrder</class>
 <widget class="QWidget" name="CreateOrder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>933</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>货到付款</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>在线支付</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="4">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>省：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QComboBox" name="cb_province">
         <item>
          <property name="text">
           <string>山西省</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>陕西省</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="2" colspan="5">
        <widget class="QLineEdit" name="le_address">
         <property name="text">
          <string>山西大学大东关校区</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>市：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QComboBox" name="cb_city">
         <item>
          <property name="text">
           <string>太原市</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>临汾市</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="6">
        <widget class="QComboBox" name="cb_region">
         <item>
          <property name="text">
           <string>杏花岭区</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>迎泽区</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>区/县：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>详细地址：</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="3">
      <widget class="QLabel" name="lb_totalprice">
       <property name="font">
        <font>
         <family>Adobe 黑体 Std R</family>
         <pointsize>15</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 0, 0);</string>
       </property>
       <property name="text">
        <string>￥1000</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>应付总额:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="5" column="0" colspan="4">
      <widget class="QPushButton" name="pushButton">
       <property name="font">
        <font>
         <family>Adobe 黑体 Std R</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 0, 0);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>提交订单</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="4">
      <widget class="QWidget" name="orderwidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <widget class="QLabel" name="lb_photo">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>300</width>
          <height>300</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>50</y>
          <width>571</width>
          <height>201</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>商品名称： </string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="lb_shoesname">
           <property name="text">
            <string>啊迪斯科解放卡戴珊覅的</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>单价</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="lb_price">
           <property name="text">
            <string>1299</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>码数</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="lb_size">
           <property name="text">
            <string>42</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>配色</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="lb_color">
           <property name="text">
            <string>黑</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>数量</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="lb_count">
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
